chainer/functions/basic_math.py

```diff
@@ -26,6 +26,27 @@ def neg(x):  # -x
     return Neg()(x)
 
 
+class Abs(function.Function):
+
+    def forward(self, x):
+        return utils.force_array(x[0].__abs__()),
+
+    def backward_cpu(self, x, gy):
+        return utils.force_array(numpy.sign(x[0]) * gy[0]),
+
+    def backward_gpu(self, x, gy):
+        gx0 = cuda.empty_like(x[0])
+        cuda.elementwise(
+            'float* gx0, const float* x0, const float* gy',
+            'gx0[i] = ((x0[i] > 0) - (x0[i] < 0)) * gy[i]',
+            'back_abs')(gx0, x[0], gy[0])
```

Comment:
Use a consistent kernel name, e.g. `abs_bwd`, following backprop kernels of other functions.